dali_mqtt_daemon.py

```diff
@@ -199,34 +198,45 @@ def on_connect(
     client.publish(
         MQTT_DALI2MQTT_STATUS.format(mqtt_base_topic), MQTT_AVAILABLE, retain=True
     )
-    lamps = dali_scan(driver_object, max_lamps)
+    lamps = dali_scan(driver_object)
+    logger.info(
+        "Found %d lamps",
+        len(lamps),
+    )
     for lamp in lamps:
         try:
-            actual_level = driver_object.send(
-                gear.QueryActualLevel(address.Short(lamp))
-            )
-            logger.debug("QueryActualLevel = %s", actual_level.value)
+            short_address = address.Short(lamp)
+            brightness_level = driver_object.send(gear.QueryActualLevel(short_address))
+
+            logger.debug("QueryActualLevel = %s", brightness_level.value)
```

Comment:
Why change ActualLevel (name according to DALI standard) ?

Reply:
To better understand that it's actually brightness level? It was in earlier review

Reply:
But lets keep the variables names in accordance to the supporting library

Reply:
If you want to